examples/driver.ts

```diff
@@ -23,6 +23,7 @@ driver.generate({
   header: '/* tslint:disable variable-name only-arrow-functions*/',
   openapiFilePath: './test/example.yaml',
   resolve: driver.compose(driver.generateFile(), driver.localResolve),
+  preserveRefPathStructure: true,
```

Comment:
The idea is to provide this boolean if we want to keep Input directory structure otherwise it should be default as we have already.